app/javascript/helpers/validate-names.js

```diff
@@ -0,0 +1,7 @@
+const validateName = (target, name) => API.get(`/api/${target}?expand=resources&attributes=name`)
+  .then(({ resources }) => resources.map((resource) => resource.name))
+  .then((results) => {
+    return results.includes(name) ? `The name "${name}" already exists in ${target}` : undefined;
```

Comment:
This line must be -

`(results.includes(name) ? sprintf(__('The name "%s" already exists in "%s"'), name, target) : undefined)`